test/core.jl

```diff
@@ -3106,3 +3106,8 @@ end
 const DATE12003 = DateTime(1917,1,1)
 failure12003(dt=DATE12003) = Dates.year(dt)
 @test isa(failure12003(), Integer)
+
+# Test error checking in bitstype
+@test_throws ErrorException bitstype 0 SPJ001
```

Comment:
When I wrote it, I didn't have a PR number yet, and I directly did a PR instead of doing an issue then PR.  Should one always do issue then PR, even if you already have a fix for a bug?